[WebGPU] Update CTS tests

tadeuzagallo opened the pull request:
#### 9c724a5a370ceefab8cfa944920c4f800d0a08c9
<pre>
[WebGPU] Update CTS tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=272604">https://bugs.webkit.org/show_bug.cgi?id=272604</a>
<a href="https://rdar.apple.com/126372152">rdar://126372152</a>

Reviewed by Mike Wyrzykowski.

Update WebGPU CTS to the latest revision using import-webgpu-cts.

* LayoutTests/http/tests/webgpu/common/framework/fixture.js:
Truncated due to commit message length, it was an auto import using import-webgpu-cts

Canonical link: <a href="https://commits.webkit.org/277493@main">https://commits.webkit.org/277493@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/858622cda0baa446693e31e0788dbe9b3211b6a3

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/47716 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/26905 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/50478 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/50399 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/43771 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/32759 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/24383 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/38835 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/48298 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/24558 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/41195 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/20132 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/22039 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/42386 "Passed tests") | [✅ 🛠 wpe-skia](https://ews-build.webkit.org/#/builders/52/builds/5765 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/44060 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/42808 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/52292 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/22752 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/19081 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/46137 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/24024 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/41349 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/45171 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/10537 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/24812 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/23744 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

mwyrzykowski commented on LayoutTests/http/tests/webgpu/common/internal/websocket_logger.js:
```diff
@@ -1,7 +1,57 @@
 /**
- * MANUALLY-GENERATED - DO NOT OVERWRITE with the webgpu-cts-import script.
```
Can we revert this file before merging please? I forgot about that

mwyrzykowski reviewed (changes requested):
Can we revert 
LayoutTests/http/tests/webgpu/common/internal/websocket_logger.js

before merging? Otherwise the output doesn't match. I forgot about that issue.

mwyrzykowski reviewed (approved):
Thank you!

tadeuzagallo committed:
[WebGPU] Update CTS tests
https://bugs.webkit.org/show_bug.cgi?id=272604
rdar://126372152

Reviewed by Mike Wyrzykowski.

Update WebGPU CTS to the latest revision using import-webgpu-cts.

* LayoutTests/http/tests/webgpu/common/framework/fixture.js:
Truncated due to commit message length, it was an auto import using import-webgpu-cts

Canonical link: https://commits.webkit.org/277493@main
